src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function adder(value) {
+    if (value !== undefined) {
+      sum += value;
+
+      return adder;
+    } else {
+      const currentSum = sum;
+
+      sum = 0;
+
+      return currentSum;
+    }
```

Comment:
else is redundant here
```suggestion
    }
    
    const currentSum = sum;

    sum = 0;

    return currentSum;
```